pkg/cmd/repo/setdefault/setdefault_test.go

```diff
@@ -498,7 +500,7 @@ func TestDefaultRun(t *testing.T) {
 				return
 			}
 			assert.NoError(t, err)
-			assert.Equal(t, tt.wantStdout, stdout.String())
+			assert.Equal(t, tt.wantStdout, stdout.String()+stderr.String())
```

Comment:
Changing the assert this way doesn't actually cover the change we're making. It will pass or fail regardless of whether or not we are printing to `stdout` or `stderr` on line 125 of `setdefault.go`. 

That's because, for this change, `stdout` is `""` while `stderr` is `"no default repository has been set; use `gh repo set-default` to select one"`. The opposite is true before we make this change. Either way, `stdout.String()+stderr.String()` will return `"no default repository has been set; use `gh repo set-default` to select one"`, so we aren't actually know if the message came from `stdout` or `stderr`.

I'd recommend extracting the logic to check `stdout` & `stderr` to their own `asserts`